api/scpca_portal/models/dataset.py

```diff
@@ -671,9 +671,28 @@ def original_file_paths(self) -> Set[Path]:
         return {Path(of.s3_key) for of in self.original_files}
 
     @property
-    def computed_file_name(self) -> Path:
-        return Path(f"{self.pk}.zip")
+    def computed_file_name(self) -> str:
+        if self.is_ccdl:
+            file_name = ccdl_datasets.COMPUTED_FILE_NAMES[self.ccdl_name]
+            return f"{file_name}.zip"
+
+        return f"{self.pk}.zip"
```

Comment:
If `self.ccdl_project_id` we should prepend that to the name with an underscore after.

If `is_ccdl` and no project id we should verify if we want to prepend `PORTAL_WIDE` or something else.